src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -546,7 +546,11 @@ private static string FormatNamespaceOrTypeSymbol(INamespaceOrTypeSymbol symbol)
 
         public override string? FormatSymbol(ISymbol? symbol)
         {
-            if (symbol == null)
+            if (symbol is null)
+                return null;
+
+            // Currently consider not finding help for a preprocessing symbol
+            if (symbol is IPreprocessingSymbol)
```

Comment:
why was this change made?

Reply:
To prevent from moving forward and calling `FormatNamespaceOrTypeSymbol`, which would operate on a null namespace if it would not exist prematurely. This check avoids this, and no NRE is thrown.

Reply:
wfm.